saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js

```diff
@@ -930,43 +964,82 @@ var initAnalysisJs = function() {
     /**********************************************************************************************/    
 
     
-    var xtickPeriodFormatters = {};
+    /**
+     * Formats the given xtick contains a period of type pl.edu.icm.saos.common.chart.value.TimePeriod
```

Comment:
`Formats the given xtick that ...` or even without that part after `contains` (`Formats the given xtick`)?